generator.rb

```diff
@@ -0,0 +1,67 @@
+# WAVE ONE/TWO
+# is_vowel? 
+def is_vowel?(letter)
+    if letter == "a" || letter == "e" || letter == "i" || letter == "o" || letter == "u"
+        return true
+    else
+        return false
+    end
```

Comment:
Your indentation in this file is set so that each indentation is 4 spaces big. In Ruby standards, indentation is best when it's 2 spaces big -- this is a setting in VS Code you can change. Let's get this configured together when we can!